.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,28 @@
-<!-- Hey there. Thank you so much for improving NewPipe. Please take a moment to fill out the following suggestion on how to structure this PR description. Having roughly the same layout helps everyone considerably :)-->
+<!-- Hey there. Thank you so much for improving NewPipe, and filling out the details. Having roughly the same layout helps everyone considerably :)-->
 
 #### What is it?
-- [ ] Bug fix (user facing)
+- [ ] Bugfix (user facing)
 - [ ] Feature (user facing)
-- [ ] Code base improvement (dev facing)
+- [ ] Codebase improvement (dev facing)
 - [ ] Meta improvement to the project (dev facing)
 
 #### Description of the changes in your PR
-<!-- While bullet points are the norm in this section, feel free to write a text instead if you can't fit it in a list -->
+<!-- While bullet points are the norm in this section, feel free to write a text instead of a list -->
 - record videos
 - create clones
 - take over the world
 
 #### Fixes the following issue(s)
-<!-- Also add reddit or other links which are relevant to your change. -->
+<!-- Also add non-reddit or other links relevant to your change. -->
 - 
 
 #### Relies on the following changes
 <!-- Delete this if it doesn't apply to you. -->
 - 
 
-#### Testing apk
-<!-- Ensure that you have your changes on a new branch which has a meaningful name. This name will be used as a suffix for the app ID to allow installing and testing multiple versions of NewPipe. Do NOT name your branches like "patch-0" and "feature-1". For example, if your PR implements a bug fix for comments, an appropriate branch name would be "commentfix". -->
+#### APK testing 
+<!-- Use a new, meaningfully named branch. The name is used as a suffix for the app ID to allow installing and testing multiple versions of NewPipe. (No names like "patch-0" and "feature-1".) "commentfix" is good if your PR implements a bugfix for comments. -->
 debug.zip
 
-#### Agreement
-- [ ] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
+#### Due dilligence
+- [ ] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).
```

Comment:
Why did you remove the last part?

Reply:
There isn't any agreement needed. This is a scourge, and should be avoided.